src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -2766,19 +2779,19 @@ private static async Task<string> CloneRestoreAndBuild(string path, Job job, str
 
             runtimeVersion = await ResolveRuntimeVersion(buildToolsPath, targetFramework, runtimeVersion, currentRuntimeVersion);
 
-            sdkVersion = await ResolveSdkVersion(sdkVersion, currentSdkVersion);
+            sdkVersion = await ResolveSdkVersion(sdkVersion, currentSdkVersion, targetFramework);
 
             aspNetCoreVersion = await ResolveAspNetCoreVersion(aspNetCoreVersion, currentAspNetCoreVersion, targetFramework);
 
             sdkVersion = PatchOrCreateGlobalJson(job, benchmarkedApp, sdkVersion);
 
             var installAspNetSharedFramework = job.UseRuntimeStore
-                || aspNetCoreVersion.StartsWith("3.0")
                 || aspNetCoreVersion.StartsWith("3.1")
```

Comment:
Remove 3.1 and 5.0 as well?

Reply:
I removed it in other parts so I assume it's logical to do it everywhere, will scan the file for these, thanks.